scripts/export_all_opslevel_yml/opslevel_yml_export_for_all_services.py

```diff
@@ -0,0 +1,81 @@
+import requests
+
+OPSLEVEL_API_TOKEN = "api_token"
+OPSLEVEL_ENDPOINT = "https://app.opslevel.com/graphql"
+
+# Function to make a GraphQL query
+def make_graphql_query(url, query, variables=None):
+    headers = {
+        'Content-Type': 'application/json',
+        'Authorization': f'Bearer {OPSLEVEL_API_TOKEN}'
+
+    }
+    data = {
+        'query': query,
+        'variables': variables
+    }
+    response = requests.post(OPSLEVEL_ENDPOINT, json=data, headers=headers)
+    return response.json()
+
+# Main function to execute the script
+def main():
+
+    # Define your GraphQL queries
+    query_1 = '''
+        query get_services($endCursor:String){
+        account{
+            services(after: $endCursor){
+            nodes{
+                name
+                id
+            }
+            pageInfo{
+                hasNextPage
+                endCursor
+            }
+            }
+        }
+        }
+    '''
+
+    query_2 = '''
+        query get_opslevel_yml($id:ID!) {
+        account {
+            configFile(id:$id) {
+            ownerType
+            yaml
+            }
+        }
+        }
+    '''
+
+    # Make the first GraphQL query to get a list of items with pagination
+    cursor = None
+    has_next_page = True
+    while has_next_page:
+        response_1 = make_graphql_query(OPSLEVEL_ENDPOINT, query_1, variables={'cursor': cursor})
+        nodes = response_1['data']['account']['services']['nodes']
+        for node in nodes:
+            node_id = node['id']
+            node_name = node['name']
```

Comment:
```suggestion
            service_alias = node['aliases'][0]
```

I think we should use the service alias because the name can have spaces, emojis and other special characters

Reply:
Good point, however getting the first alias in the list isn't a guarantee either that I'll get the "right" alias either.

It would be great if we could query for the "primary" alias, as there may be cases where the first alias isn't the "primary" alias, for example:

```
            "name": "Book Locator Service",
            "aliases": [
              "Book Locator Service",
              "book-locator-service",
              "book_locator_service"
            ],
```